media/css/protocol/protocol-mozilla.scss

```diff
@@ -61,3 +61,9 @@ $image-path: '/media/protocol/img';
     padding: $spacing-sm;
     margin-bottom: $spacing-xl;
 }
+
+
+// hide <template> elements in the DOM.
+template {
+    display: none !important; /* stylelint-disable-line declaration-no-important */
+}
```

Comment:
Question for @craigcook: Do we need to add any of this to Protocol going forward?

Reply:
I can certainly see uses for `<template>` in Protocol, for example the modal component could use it instead of putting content directly in the DOM. Adding this rule here just so older browsers don't render it seems worthwhile too.